Tests/SwiftProtobufTests/Test_Naming.swift

```diff
@@ -184,3 +184,165 @@ class Test_FieldNamingInitials: XCTestCase {
     msg.clearTheURL()
   }
 }
+
+class Test_ExtensionNamingInitials_MessageScoped: XCTestCase {
+  func testLowers() {
+    var msg = SwiftUnittest_Names_ExtensionNamingInitials()
+
+    msg.SwiftUnittest_Names_Lowers_http = 1
```

Comment:
So these are the interesting cases at the moment.  If you compare this to the pain fields (above).  There the "http" comes first, so lower make sense, but here because it is always scoped with the message that defined it, should we just go to uppercase for the "http" at the end?  Notice whatis looks like on the `has` and `clear` below also.